Marlin/Configuration.h

```diff
@@ -1310,8 +1310,8 @@
   #endif
 
   #if ANY(KNUTWURST_4MAX, KNUTWURST_4MAXP2)
-    #define DEFAULT_XJERK  8.0
-    #define DEFAULT_YJERK  8.0
+    #define DEFAULT_XJERK  8.2
```

Comment:
With DirectDrive on v2.0 this value is too high

Reply:
Values comes from official Anycubic 4MaxPro1 GitHub repo:

https://github.com/ANYCUBIC-3D/4MAX_PRO/blob/2b0c575df99d2c76bfd3b41e14752763350c7317/Configuration.h#L544